qiskit/circuit/library/standard_gates/x.py

```diff
@@ -437,7 +437,7 @@ def __init__(self, label=None, ctrl_state=None, *, angle=None):
             angle (float): DEPRECATED. The angle used in the controlled-U1 gates. An angle of π/8
                 yields the sqrt(X) gates, an angle of π/4 the 3-qubit controlled X gate.
         """
-        super().__init__('c3sx', 4, [], num_ctrl_qubits=3, label=label,
+        super().__init__('c3sqrtx', 4, [], num_ctrl_qubits=3, label=label,
```

Comment:
Can we keep the name `c3sx`? Since the square-root of X gate is called SX in Qiskit this would be more consistent.

Reply:
I have no problem of using `c3sx`, in fact I prefer this name as well, but in the gate is called `c3sqrtx` in `qelib1.inc`: https://github.com/Qiskit/qiskit-terra/blob/0c8bb3dbf8d688590431ca79a83ba8aede84ed20/qiskit/qasm/libs/qelib1.inc#L242. It's also present here: https://github.com/Qiskit/qiskit-terra/blob/0c8bb3dbf8d688590431ca79a83ba8aede84ed20/qiskit/qasm/pygments/lexer.py#L56

Reply:
Oh, good point. Maybe we can change it in `qelib1.inc`, so that we are consistent overall 🙂